packages/openfeature-extra/src/lib/hooks/open-telemetry-hook.ts

```diff
@@ -1,18 +1,27 @@
-import { FlagValue, Hook, HookContext } from '@openfeature/openfeature-js';
+import {
+  FlagEvaluationDetails,
+  FlagValue,
+  Hook,
+  HookContext,
+} from '@openfeature/openfeature-js';
 import { Span, trace, Tracer } from '@opentelemetry/api';
 
 export const SpanProperties = {
-  FEATURE_FLAG_CLIENT_NAME: 'feature_flag_client_name',
-  FEATURE_FLAG_CLIENT_VERSION: 'feature_flag_client_version',
-  FEATURE_FLAG_SERVICE: 'feature_flag_service',
-  FEATURE_FLAG_ID: 'feature_flag_id',
-  FEATURE_FLAG_VALUE: 'feature_flag_value',
+  FLAG_KEY: 'feature_flag.flag_key',
+  CLIENT_NAME: 'feature_flag.client.name',
+  CLIENT_VERSION: 'feature_flag.client.version',
+  PROVIDER_NAME: 'feature_flag.provider.name',
+  PROVIDER_MANAGEMENT_URL: 'feature_flag.provider.management_url',
+  VARIANT: 'feature_flag.evaluated.variant',
+  VALUE: 'feature_flag.evaluated.value',
```

Comment:
Definitely interested in any feedback on the OTel semantics.

Reply:
https://github.com/open-feature/spec/issues/30